.pre-commit-config.yaml

```diff
@@ -9,7 +9,8 @@ repos:
         entry: pylint
         language: system
         types: [python]
-        files: "^express/"
+        files: "^express/\n
+                |^tests/"
```

Comment:
Any reason to go with this instead of what I proposed?
My proposal came from the [pre-commit docs](https://pre-commit.com/#regular-expressions) and I think it's more readable and easier to extend.

Reply:
I switched it over to the regex approach, the regex statement did not work initially so that's why I opted for the string approach. But then I modified it a bit and now it works.